code/modules/events/blob.dm

```diff
@@ -12,7 +12,7 @@
 	if(!T)
 		return kill()
 
-	var/list/candidates = pollCandidates("Do you want to play as a blob infested mouse?", ROLE_BLOB, 1)
+	var/list/candidates = SSghost_spawns.poll_candidates("Do you want to play as a blob infested mouse?", ROLE_BLOB, TRUE, source = /mob/living/simple_animal/mouse/blobinfected)
```

Comment:
Any reason youre not just doing `source=src` here?

Reply:
`source` here refers to the mob prototype/instance to display as an icon in the alert panel. Passing an (event) datum makes no sense because it has no icon

Reply:
Ah I see